test-results/2026-7-1-test-report.md

```
@@ -0,0 +1,44 @@
+## 指标  ATE RMSE / median / max
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)